### PR TITLE
nanopolish: upgrade to v0.10.1

### PR DESCRIPTION
**Pull request by suujia**
- [ ] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)? 
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change? 
- [ ] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting? 
- [ ] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

### CODE REVIEW

**Comment by sjackman** [comment]
You have merge commit in this PR. Can you please remove the merge commit and rebase onto `master`? You can do that like so…
```sh
git checkout -B nanopolish origin/master
git cherry-pick 4b674d9f038cd0c00fcfc7534958b0b57fb9076b
git push suujia nanopolish --force
```

File: Formula/nanopolish.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Nanopolish < Formula
   desc "Signal-level algorithms for MinION data"
   homepage "https://github.com/jts/nanopolish"
   url "https://github.com/jts/nanopolish.git",
-        :tag => "v0.9.0",
-        :revision => "1c6ae110a1b7d0ca5072025b1889997fec0828ed"
+        :tag => "v0.10.1",
+        :revision => "0438b85dd3acf84a34c8e3d2098d55ab040900e3"
   revision 1
```
Comment (sjackman) [inline Formula/nanopolish.rb]: Please remove `revision 1` when moving to a new upstream version.

**Comment by suujia** [comment]
Reopen, closed by accident.

**Comment by sjackman** [comment]
Merged. Thanks, Susie! 🍍

**Pull request closed without merging**